src/auth/src/lib.rs

```diff
@@ -41,3 +41,5 @@ pub mod credentials;
 ///
 /// [Tokens]: https://cloud.google.com/docs/authentication#token
 pub mod token;
+
+pub(crate) type Result<T> = std::result::Result<T, crate::errors::CredentialError>;
```

Comment:
Consider a comment, in case this becomes public.  Something like:

https://docs.rs/reqwest/latest/reqwest/type.Result.html

Reply:
Done.